habitat_baselines/config/default.py

```diff
@@ -193,5 +203,14 @@ def get_config(
         config.CMD_TRAILING_OPTS = config.CMD_TRAILING_OPTS + opts
         config.merge_from_list(config.CMD_TRAILING_OPTS)
 
+    if config.NUM_PROCESSES != -1:
+        warnings.warn(
```

Comment:
Once again, any reason we aren't using the habitat logger here?

Reply:
Same reason